.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = ./.git
+ignore-words = .codespell/ignore-words.txt
+exclude-file = .codespell/exclude-file.txt
```

Comment:
Is this still needed, or was it just used for testing?

Reply:
There are multiple ways to silence false positives:
* ignore words (option `ignore-words`)
* ignore whole lines (option `exclude-file`)